index.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Microverse Technical Document</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+  <nav id="navbar">
+    <header><h1>JS Documentation</h1></header>
+    <a href="#Introduction" class="nav-link">Introduction</a>
+    <a href="#What_you_should_already_know" class="nav-link">What you should already know</a>
+    <a href="#JavaScript_and_Java" class="nav-link">Javascript and Java</a>
+    <a href="#Hello_world" class="nav-link">Hello World</a>
+    <a href="#Variables" class="nav-link">Variables</a>
+    <a href="#Declaring_variables" class="nav-link">Declaring Variables</a>
+    <a href="#Variable_scope" class="nav-link">Variable Scope</a>
+    <a href="#Global_variables" class="nav-link">Global Variables</a>
+    <a href="#Constants" class="nav-link">Constants</a>
+  </nav>
+
+  <main id="main-doc">
+    <section class="main-section" id="Introduction">
+      <header><h2>Introduction</h2>
+      </header>
+      <p>JavaScript is a cross-platform, object-oriented scripting language. It is a small and lightweight language. Inside a host environment (for example, a web browser), JavaScript can be connected to the objects of its environment to provide programmatic control over them.</p>
+      <br>
+      <p>JavaScript contains a standard library of objects, such as Array, Date, and Math, and a core set of language elements such as operators, control structures, and statements. Core JavaScript can be extended for a variety of purposes by supplementing it with additional objects; for example:</p>
+
+    <ul>
+      <li>Client-side JavaScript extends the core language by supplying objects to control a browser and its Document Object Model (DOM). For example, client-side extensions allow an application to place elements on an HTML form and respond to user events such as mouse clicks, form input, and page navigation.</li>
+
+      <li>Server-side JavaScript extends the core language by supplying objects relevant to running JavaScript on a server. For example, server-side extensions allow an application to communicate with a database, provide continuity of information from one invocation to another of the application, or perform file manipulations on a server. </li>
+    </ul>
+
+    </section>
+
+      <section class="main-section" id="What_you_should_already_know">
+        <header><h2>What you should already know</h2></header>

```

Comment:
- [ ]  Please be more polite, professional, friendly and transparent in your comments while doing code reviews. Your comment looks like you are ordering and do have not a strong reason to ask for adding something to the document.